doc/repmgr.xml

```diff
@@ -26,7 +26,7 @@
   <abstract>
    <para>
    This is the official documentation of &repmgr; &repmgrversion; for
-   use with PostgreSQL 9.4 - PostgreSQL 15.
+   use with PostgreSQL 9.4 - PostgreSQL 16.
```

Comment:
If we remove 9.4, 9.5, 9.6 from the previous file, then we must change 9.4 to 10 here too

Reply:
Correct. And I'd have to change that in the website as well. Let me push one more commit